src/devices/Mpu9250/README.md

```diff
@@ -27,7 +27,7 @@ var mpui2CConnectionSettingmpus = new I2cConnectionSettings(1, Mpu9250.DefaultI2
 using Mpu9250 mpu9250 = new Mpu9250(I2cDevice.Create(mpui2CConnectionSettingmpus), i2CDeviceAk8963: I2cDevice.Create(new I2cConnectionSettings(1, Ak8963.DefaultI2cAddress)));
 ```
 
-You can find an example in the [sample](./samples/Mpu9250.sample.cs) directory. Usage is straightforward including the possibility to have a calibration for all sub sensors.
+You can find an example in the [sample](./samples/Program.cs) directory. Usage is straightforward including the possibility to have a calibration for all sub sensors.
```

Comment:
same question about copying .cs files - consider removing this link

Reply:
They are transformed like this:
![image](https://user-images.githubusercontent.com/8145578/127995702-855b56f3-9c17-4b74-86ba-43a53105f57a.png)